src/xGadgetron/cGadgetron/tests/CMakeLists.txt

```diff
@@ -52,7 +52,7 @@ if(EXE_ISMRMRD_SL AND RERUN_ISMRMRD_SHEPP_LOGAN)
         COMMAND rm -f ${TESTDATA_OUTPUT_NAME_FULL}
         COMMAND rm -f ${TESTDATA_OUTPUT_NAME_GRAPPA}
         COMMAND env LD_LIBRARY_PATH=${CMAKE_INSTALL_PREFIX}/lib:$LD_LIBRARY_PATH ${EXE_ISMRMRD_SL} -r 2 -o "${TESTDATA_TEMP_PREFIX}_1.h5"
-        COMMAND env LD_LIBRARY_PATH=${CMAKE_INSTALL_PREFIX}/lib:$LD_LIBRARY_PATH ${EXE_ISMRMRD_SL} -a 2 -o "${TESTDATA_TEMP_PREFIX}_2.h5"
+        COMMAND env LD_LIBRARY_PATH=${CMAKE_INSTALL_PREFIX}/lib:$LD_LIBRARY_PATH ${EXE_ISMRMRD_SL} -w 64 -a 2 -o "${TESTDATA_TEMP_PREFIX}_2.h5"
```

Comment:
@evgueni-ovtchinnikov is this the identical command that you had used to create the data whose values are now stored in the test.txt files?

Because strangely now the tests fail with a larger deviation than before:
```
ValueError: +++ test 0 failed: expected 148, got 209.3 (tolerance 0.296): 
```
(https://github.com/SyneRBI/SIRF/pull/936/checks#step:7:945)

Reply:
@johannesmayer no, it appears these files were actually created by somebody else (Christoph? David?)

Just checked with the Shepp-Logan generator source: confusingly, it generates r*a repetitions, where r is the value requested by the user via option -r and a is the acceleration factor requested via option -a.

So you generated data of double size, and 148*sqrt(2) = 209.

This means we will have to either remove the second repetition after generating the undersampled data or re-run all tests with --record to get correct txt files. Can you try the first option please? I believe you have all the necessary tools in your C++ contributions to SIRF.

Reply:
> @johannesmayer no, it appears these files were actually created by somebody else (Christoph? David?)
> 
> Just checked with the Shepp-Logan generator source: confusingly, it generates r*a repetitions, where r is the value requested by the user via option -r and a is the acceleration factor requested via option -a.
> 
> So you generated data of double size, and 148*sqrt(2) = 209.
> 
> This means we will have to either remove the second repetition after generating the undersampled data or re-run all tests with --record to get correct txt files. 

A question is of course how the original data were generated. It seems they were checked in by @casperdcl but it's unlikely he generated them.

> Can you try the first option please? I believe you have all the necessary tools in your C++ contributions to SIRF.

I see a few options:
- re-record the numbers, which means we always have to `RERUN...=ON`
- remove the second repetition as suggested by @evgueni-ovtchinnikov. I think this creates a fair amount of work. 
- make the test independent of the number of repetitions by testing on the relative norm of the steps.

I prefer the 3rd option, as this might make it also less sensitive to future changes

Reply:
I've implemented this now. I've tested it on my local VM with both original and newly generated files. Both work fine.

Reply:
> A question is of course how the original data were generated. It seems they were checked in by @casperdcl but it's unlikely he generated them.

Which files are you referring to?

Reply:
@casperdcl this is the commit https://github.com/SyneRBI/SIRF_data/commit/718a71164516326857916c4b433cad6c795e053f#diff-38ea6924eefb5a45710d325761386323a3e7694c9a1697d0ca26f1ae4e8b26bd

Reply:
Ah yes. I remember we moved all the binary data out of SIRF's history (to a `data` submodule) to reduce the repo size - so there's no record of the original commits. I don't have the original repo from ~5 years ago so can't tell :/